eval/action.yml

```diff
@@ -0,0 +1,66 @@
+name: "Sandard Eval"
+description: "Evaluate Standard based projects"
+
+inputs:
+  key_path:
+    description: "The path to place the Nix secret Key"
+    required: true
```

Comment:
```suggestion
    required: false
```

Reply:
ah yeah, this is probably a little too opinionated :sweat_smile: